canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx

```diff
@@ -380,7 +380,7 @@ class VirtualizedTable extends React.Component {
 		}
 
 		if (this.props.selectable) {
-			const rowSelected = this.isRowSelected(rowData.originalRowIndex);
+			const rowSelected = this.props.sortDirection ? this.isRowSelected(rowData.index) : this.isRowSelected(rowData.originalRowIndex); // use current row index when Sorted
```

Comment:
@caritaou Do you see any issue with this change?  I did some testing around row selection and for the most part it doesn't really work correctly onSort in the table controls.  It just clears the selection.

Reply:
but this does fix the issue in the expressionBuilder around sorting and selection.

Reply:
This looks good to me, considering this never worked for sorting columns with row selections.